ios/RCTTWVideoModule.m

```diff
@@ -70,7 +70,7 @@ @interface RCTTWVideoModule () <TVIRemoteDataTrackDelegate, TVIRemoteParticipant
 @property (strong, nonatomic) TVIAppScreenSource *screen;
 @property (strong, nonatomic) TVILocalParticipant* localParticipant;
 @property (strong, nonatomic) TVIRoom *room;
-@property (strong, nonatomic) NSString *customVideoTrackName;
+@property (strong, nonatomic) NSString *videoTrackName;
```

Comment:
❔ should we go ahead and rename this variable to `localVideoTrackName` across both platforms to be more specific?

Reply:
So unfortunately the `@property` annotation [creates its own](https://stackoverflow.com/a/2616596/194065) `setVideoTrackName()`, and since I don't know Obj-C well enough, I don't know if it's possible to export this generated setter to RN.  When I had the exported setter matching this property name, I got a crash on the setter, presumably due to the conflict in function names.  Thus, I kept the `@property` as `videoTrackName` in both iOS and Android, and let all the javascript use `localVideoTrackName`.  Not the best, but this was my reasoning.  Open to other ideas of course though.